run make test on arm64 machine always failed

## Bug Report

### What version of TiKV are you using?
branch master

### What operating system and CPU are you using?
centos7.6 on arm64(Kunpeng 920)

### Steps to reproduce
`make test`

### What did you expect?
No error
### What did happened?
```shell
stack backtrace:
test s3::tests::test_s3_storage_with_virtual_host ... ok
test kms::tests::test_aws_kms ... ok
test s3::tests::test_s3_config ... ok
   0: rust_begin_unwind
             at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/panicking.rs:50:5
   3: aws::s3::tests::test_s3_storage
             at ./src/s3.rs:569:9
   4: aws::s3::tests::test_s3_storage::{{closure}}
             at ./src/s3.rs:528:5
   5: core::ops::function::FnOnce::call_once
             at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/ops/function.rs:227:5
   6: core::ops::function::FnOnce::call_once
             at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
test s3::tests::test_s3_storage ... FAILED

failures:

failures:
    s3::tests::test_s3_storage

test result: FAILED. 6 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.18s

error: test failed, to rerun pass '-p aws --lib'
make[1]: *** [Makefile:262: run] Error 101
make[1]: Leaving directory '/home/jenkins/tikv'
make: *** [Makefile:270: test] Error 2

```

Log datail is here https://ci.pingcap.net/blue/organizations/jenkins/tikv_nightly_test_arm64/detail/master/16/pipeline/

--- a/components/cloud/aws/src/s3.rs
+++ b/components/cloud/aws/src/s3.rs
@@ -525,6 +525,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(feature = "failpoints")]
     fn test_s3_storage() {
         let magic_contents = "5678";
         let bucket_name = StringNonEmpty::required("mybucket".to_string()).unwrap();

--- a/components/cloud/aws/src/s3.rs
+++ b/components/cloud/aws/src/s3.rs
@@ -524,8 +524,8 @@ mod tests {
         assert!(S3Storage::new(config).is_err());
     }
 
-    #[test]
     #[cfg(feature = "failpoints")]
+    #[test]
     fn test_s3_storage() {
         let magic_contents = "5678";
         let bucket_name = StringNonEmpty::required("mybucket".to_string()).unwrap();

